009_team_7_agent_design/009_team_7_agent_design.tex

```diff
@@ -1 +1,443 @@
+
+The flow chart in \Cref{fig: operation flow chart} outlines the decisions made concerning how much food to take.
+
+\begin{figure}[h!]
+    \begin{center}
+        \includegraphics[scale=0.75]{009_team_7_agent_design/Images/v2.drawio.png}
+    \end{center}
+    \caption{Food taking decision flow chart.}
+    \label{fig: operation flow chart}
+\end{figure}
+
+Upon completion of the memory bank updates and risk assessment routines, the agent then enters its food consumption routine. This routine runs only once per day when the platform is on the agent's floor. The routine begins by assessing whether the agent's health has been critical for multiple days, if this is the case the agent overrides any requests in effect and prioritises getting its health out of critical (Note however no treaties that prevent the agent from leaving critical HP would be accepted by the agent and thus no treaties would be broken at this point, treaty conditions are detailed in a later section). The kindness adjuster is weighted down and the TakeFood function is executed. If the agent's health is not critical the routine evaluates whether it can abide within the constraints imposed for accepted treaties and requests. Once the constraints are in place the TakeFood function is executed in accordance to the greed and kindness adjusters within the bounds of the request/treaty constraints. 
+
+Once the routine is complete the seenPlatform variable is reset to indicate the end of the day.
+
+\subsection{Messages}
+\label{subsec: Messages}
+Although not a crucial feature of the final agent strategy - reasonable consideration went into deciding how to leverage messaging to better serve an agent during a simulation of the tower.
+
+In order to think about how messaging could benefit the agent, two scenario's can be considered - one where the agent is sending messages to make an alert, enquiry or request and the converse, where an agent receives messages and can perform some action based on the incoming information or request.
+\subsubsection{Sending Messages}
+\label{subsubsec: Sending Messages}
+The agent's behaviour is majorly determined by the intrinsic characteristics that constitute the agent's personality and so it was deemed that sending messages to gather information served no significant additional purpose as the new information would not have any influence on the next decision made. It is, however, important to benefit the collective effort of maintaining survival as a population - therefore an agent will send a reply to acknowledge information from other agent's and respond to requests stating whether they can comply.
+
+\subsubsection{Receiving Messages}
+\label{subsubsec: Receiving Messages}
+The agent may adapt its behaviour upon receipt of request messages from other agent's. The criteria that must be met in order to comply with a request is as follows; 
+
+\begin{itemize}
+  \item The agent's extraversion rating must be higher than 5
```

Comment:
```suggestion
  \item The agent's extraversion rating must be higher than 5.
```

Reply:
you shoudl probably have a comma at the end of the sentence, even if it is in a list